source/draw/RasterRgb.ooc

```diff
@@ -162,7 +160,7 @@ RasterRgb: class extends RasterPacked {
 		if (original instanceOf(This))
 			result = (original as This) copy()
 		else {
-			result = This new(original)
+			result = This new(original size)
```

Comment:
When the copied image was not packed, padding would be removed to handle sub-images.
When the copied image was packed, padding was preserved.

To remove both constructors, both cases will have to remove the padding to be on the safe side with YUV images.